tests/test_dummy.py

```diff
@@ -1,6 +0,0 @@
-from django.test import TestCase
-
-
-class TestDummy(TestCase):
-    def test_one_plus_one(self):
-        assert 1 + 1 == 2
```

Comment:
This test seemed rather useless 😛 but I can restore it if there's any reason.

Reply:
Yeah, this one I put on new projects to be sure that the test infra is collecting the tests correctly. Can be removed.